src/models/logic/transaction.ts

```diff
@@ -670,3 +822,7 @@ export async function getRegularKeyOwnerByPublicKey(
     }
     return tx.get("signer");
 }
+
+export function calculateTxEvaluatedKey(tx: TransactionAttribute) {
```

Comment:
Didn't we use `create...` instead of `calculate`? Do you prefer this way?

Reply:
I'll change it to use `create`.

Reply:
I fixed it.